backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/handlers/EditingContextRepresentationDescriptionsEventHandler.java

```diff
@@ -70,9 +74,14 @@ public boolean canHandle(IEditingContext editingContext, IInput input) {
 
     @Override
     public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDescriptionSink, IEditingContext editingContext, IInput input) {
+        var object = Optional.empty();
```

Comment:
`object` => `optionalObject`. The name `object` is also used as a parameter to `findAllCompatibleRepresentationDescriptions` but its ambiguous because on on is the `Optional` and the other is its value (if present).

Reply:
Fixed in https://github.com/eclipse-sirius/sirius-components/commit/e600eaf4eaade6cee3385e0950f3e6130d8ac056